### PR TITLE
[US-22] Horizontal bar chart

### PR DESCRIPTION
**Pull request by 3ace**
This PR includes changes available in PR #5, so it's better to merge that PR first

### CODE REVIEW

**Review by sampila — APPROVED** [review]
Looks good to me

File: examples/horizontal_bar_chart/preview.png
Comment (gunnsth) [inline examples/horizontal_bar_chart/preview.png:1]: axis missing in rendering?  Do we need to set default stroke width like in the previous PR?

Comment (3ace) [reply examples/horizontal_bar_chart/preview.png:1]: yes, this should've been fixed in the latest updates.

**Review by adrg — APPROVED** [review]
Looks good. Horizontal bar charts are a nice addition to the library.

Did you also regenerate the vertical bar chart examples to check if there are no inadvertent changes?

**Comment by 3ace** [comment]
@adrg yes, I rebuild all the examples to see if the other are affected by this. That's why I have create a shell script to rebuild all the examples included in my previous PR to help this.

**Review by gunnsth — APPROVED** [review]
LGTM

**Pull request merged**